sources/api/apiclient/src/exec/connect.rs

```diff
@@ -1,9 +1,13 @@
 //! The 'connect' module provides a function for connecting to a WebSocket over a Unix-domain
 //! socket, which is a bit more finicky than normal.
 
+use base64::{engine, Engine as _};
```

Comment:
Do we need `Engine` in scope for a trait or something? Not sure I follow what this does.

Reply:
Yes, we need `Engine` in scope otherwise we can't call the methods to generate the base64 strings.